ledger/src/blockstore_db.rs

```diff
@@ -1016,13 +1014,28 @@ pub struct WriteBatch<'a> {
     map: HashMap<&'static str, &'a ColumnFamily>,
 }
 
+pub struct BlockstoreOptions {
+    pub access_type: AccessType,
+    pub recovery_mode: Option<BlockstoreRecoveryMode>,
+    pub enforce_ulimit_nofile: bool,
+}
+
+impl Default for BlockstoreOptions {
+    /// The default options are the one used by [`Blockstore::open`].
+    /// Currently, it is set to `AccessType::PrimaryOnly`, no recovery
+    /// with unlimited number of open files.
```

Comment:
nit: "The default options are the ~~one~~ **values** used by ..." (make plurality match, and change "ones" to "values")

Also, I think we can probably drop the lines stating what default values are; the function body is fairly self-documenting in this regard

Reply:
> I think we can probably drop the lines stating what default values are; the function body is fairly self-documenting in this regard

The repetitive is mainly for those who happen to only read documents from the generated html files, but I guess we probably don't have such group of people at this moment :p.